adapters/libuv.h

```diff
@@ -90,7 +90,7 @@ static void on_handle_close(uv_handle_t *handle) {
     // else, wait for `on_timer_close`
 }
 
-#if UV_MAJOR_VERSION > 0 || UV_MINOR_VERSION > 11 || (defined(UV_VERSION_PATCH) && UV_VERSION_PATCH > 22)
+#if !defined(UV_VERSION_PATCH) || (UV_MAJOR_VERSION < 1 && UV_MINOR_VERSION < 12 && UV_VERSION_PATCH < 23)
 // libuv removed `status` parameter since v0.11.23
```

Comment:
This is maybe wrong for versions like 0.10.37 (it exists).

minor < 12 doesn't always mean minor == 11.

Reply:
Oh, thanks for the note, you are correct, `major.minor.patch < 0.11.23` expands to something like `major < 0 || (major == 0 && minor < 11) || (minor == 11 && patch < 23)`

Reply:
Let me try reduce it, could you lend some help?

Reply:
Major < 0 can't happen so this part can be removed.

I think `major == 0 && (minor < 11 || (minor == 11 && patch < 23))`